pkg/controllers/deprovisioning/drift.go

```diff
@@ -66,7 +65,8 @@ func (d *Drift) filterAndSortCandidates(ctx context.Context, nodes []*Candidate)
 		return nil, fmt.Errorf("filtering candidates, %w", err)
 	}
 	sort.Slice(candidates, func(i int, j int) bool {
-		return machineutil.GetDriftedTime(candidates[i].Machine).Before(machineutil.GetDriftedTime(candidates[j].Machine))
+		return candidates[i].Machine.StatusConditions().GetCondition(v1alpha5.MachineDrifted).LastTransitionTime.Inner.Time.Before(
```

Comment:
Isn't this the same as what it currently is? 

https://github.com/aws/karpenter-core/blob/main/pkg/utils/machine/machine.go#L321-L326

Reply:
It is, I just didn't see the need for the separate utils func since we know that the status condition is set at this point.